lein-template/resources/leiningen/new/duct/base/config.edn

```diff
@@ -5,19 +5,16 @@
   :duct.router/ataraxy
   {:routes {<<#example?>>[:get "/example"] [:<<namespace>>.handler/example]<</example?>>}}<</ataraxy?>><<#web?>><<^ataraxy?>>
 
-  :duct.router/cascading
-  [<<#example?>>#ig/ref :<<namespace>>.handler/example<</example?>>]<</ataraxy?>><</web?>><<#example?>>
+  :duct.router/cascading <<&cascading-routes>><</ataraxy?>><</web?>><<#example?>>
```

Comment:
If `cascading-routes` can have a new line, then this will result in a trailing space. Consider instead:

```clojure
:duct.router/cascading<<&cascading-routes>>
```